bing_bert/bert_large_lamb_nvidia_data.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "bing_bert_large_lamb_seq",
+    "bert_token_file": "bert-large-uncased",
+    "bert_model_file": "bert-large-uncased",
+    "bert_model_config": {
+        "vocab_size_or_config_json_file": 119547,
```

Comment:
Curious, how this `vocab_size` is 119k with NVIDIA's dataset, whereas, in NVIDIA's own config, it is 30k. 
https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/LanguageModeling/BERT/bert_config.json#L12

Reply:
This is a good question. In reality, `vocab_size_or_config_json_file` is not used for computing vocabulary size, Instead, the vocabulary is computed from the tokenizer as seen [here](https://github.com/microsoft/DeepSpeedExamples/blob/47766e0d7758a53c2582e517ae1c930bcc012518/bing_bert/turing/models.py#L113-L118). For confirmation you can examine the log for the output of this [print statement](https://github.com/microsoft/DeepSpeedExamples/blob/47766e0d7758a53c2582e517ae1c930bcc012518/bing_bert/turing/models.py#L118) to see the following: 

`worker-0: VOCAB SIZE: 30528`